products/dialogflow/api.yaml

```diff
@@ -129,4 +129,129 @@ objects:
         values:
           - :TIER_STANDARD
           - :TIER_ENTERPRISE
-          - :TIER_ENTERPRISE_PLUS
\ No newline at end of file
+          - :TIER_ENTERPRISE_PLUS
+  - !ruby/object:Api::Resource
+    name: 'Intent'
+    base_url: "projects/{{project}}/agent/intents/"
+    self_link: "{{name}}"
```

Comment:
I'm confused what the value of 'name' is. I assumed based on the import format of `"projects/{{project}}/agent/intents/{{name}}"` that name would just be a uuid or some such unique identifier. But if that is the case then I don't think the self_link is `https://dialogflow.googleapis.com/v2/{{a unique id}}`

fyi I think by default self_link is {{base_url}}/{{name}